tests/test_all_fields.py

```diff
@@ -79,6 +79,10 @@ def test_run(self):
                 # As we can't generate following field by activecampaign APIs and UI, so removed it form expectation list.
                 if stream == "ecommerce_orders":
                     expected_all_keys = expected_all_keys - {'order_products'}
+                elif stream == "contacts":
+                    expected_all_keys = expected_all_keys - {'email_empty'}
+                elif stream == "users":
+                    expected_all_keys = expected_all_keys - {'password_updated_utc_timestamp', 'cdate', 'udate'}
```

Comment:
```suggestion
                expected_all_keys = expected_all_fields - self.MISSING_FIELDS.get(stream, set())
```
This looks bit dirty. you can define `MISSING_FIELDS` as dict with `key` as `stream name` and `value` as `set of missing fields`